test/lit.cfg

```diff
@@ -752,7 +752,7 @@ if run_vendor == 'apple':
     extra_frameworks_dir = make_path(config.variant_sdk, "..", "..", "..",
                                      "Developer", "Library", "Frameworks")
     target_options = (
-        "-target %s %s %s" %
+        "-target %s %s %s -ignore-module-source-info" %
         (config.variant_triple, resource_dir_opt, mcp_opt))
     target_options_for_mock_sdk = (
         "-target %s %s %s" %
```

Comment:
Oops, why isn't this based on `target_options`. 😭 It belongs here too, *and* in the non-Apple sections!